In [9]:
import h5py
import mne
from eegatscale.models import LinearHeadBENDR
from scripts.finetune_cv import H5PYDatasetLabeled
from scripts.finetune_cv_nogroups import H5PYDatasetLabeled as H5PYDatasetLabeledNoGroups
from torch.utils.data import DataLoader
from eegatscale.transforms import StandardizeLabel, Standardize
import numpy as np
import torch
import pickle
import os
# from dataset_utils import PickleDataset
from tqdm import tqdm
from glob import glob
import warnings
warnings.filterwarnings("ignore")

# TUSZ data

In [10]:
number = 2
# tusz_model_dir = f"/scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_{number}/"
# tusz_model_dir = f"/scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset/"

# whole dataset split train
tusz_model_dir = "/scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train"

models = glob(f"{tusz_model_dir}/*/*/*.ckpt")

# we need to test all the models on the eval set
data_dir_eval = "/scratch/s194101/data/preprocessed_downstream/tusz/noica_60.0_titans_combine_eval_combined"
data_dir_train = "/scratch/s194101/data/preprocessed_downstream/tusz/noica_60.0_titans_combine_whole_dataset_combined"
data_dir_split_train = "/scratch/s194101/data/preprocessed_downstream/tusz/noica_60.0_titans_combine_whole_dataset_split_train_combined"
data_dir_split_eval = "/scratch/s194101/data/preprocessed_downstream/tusz/noica_60.0_titans_combine_whole_dataset_split_eval_combined"
transform = StandardizeLabel()


data_eval = H5PYDatasetLabeledNoGroups(data_dir_eval, transform=transform)
# data_eval = H5PYDatasetLabeledNoGroups(data_dir_train, transform=transform)
# data_eval = H5PYDatasetLabeledNoGroups(data_dir_split_train, transform=transform)
# data_eval = H5PYDatasetLabeledNoGroups(data_dir_split_eval, transform=transform)
print(f"{len(data_eval)} samples in eval set")



dataloader_eval = DataLoader(data_eval, batch_size=80, shuffle=True)

872 samples in eval set


In [11]:
# x, y = next(iter(dataloader_eval))
# print(x.shape, y.shape)

# y_pred = model(x)

# y_pred.argmax(dim=1), y

# # y_pred.argmax(dim=1) == y

In [12]:
best_acc = 0
best_sem = 0
best_model = None

for model_path in tqdm(models):
    print(f"Testing model {model_path}")
    model = LinearHeadBENDR(model_path, encoder_h=512, in_features=19, out_features=2)
    state_dict = torch.load(model_path, map_location=torch.device('cpu'))["state_dict"]
    model.load_state_dict(state_dict)
    model.eval()
    print("model loaded")


    # test model on eval set
    accs = []
    with torch.no_grad():
        for i, batch in enumerate(dataloader_eval):
            x, y = batch
            y_pred = model(x)
            batch_accuracy = (y_pred.argmax(dim=1) == y).float().mean().item()
            accs.append(batch_accuracy)
            # print(f"Batch {i}, accuracy: {batch_accuracy}")
        
        # standard error of the mean
        accs = np.array(accs)
        sem = np.std(accs) / np.sqrt(len(accs))


        print(f"Model {model_path} accuracy on eval set: {np.mean(accs)} +- {sem}")
        if np.mean(accs) > best_acc:
            best_acc = np.mean(accs)
            best_sem = sem
            best_model = model_path
        
        print(f"Best model so far: accuracy: {best_acc} +- {best_sem} | {best_model}")


  0%|          | 0/50 [00:00<?, ?it/s]

Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
model loaded


  2%|▏         | 1/50 [00:18<15:12, 18.62s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt accuracy on eval set: 0.7361111044883728 +- 0.008454882142943827
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_41/checkpoints/epoch=12-step=481.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_41/checkpoints/epoch=12-step=481.ckpt
model loaded


  4%|▍         | 2/50 [00:35<13:52, 17.33s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_41/checkpoints/epoch=12-step=481.ckpt accuracy on eval set: 0.7271464683792808 +- 0.01773112411852235
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_17/checkpoints/epoch=7-step=296.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_17/checkpoints/epoch=7-step=296.ckpt
model loaded


  6%|▌         | 3/50 [00:51<13:20, 17.03s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_17/checkpoints/epoch=7-step=296.ckpt accuracy on eval set: 0.731186872178858 +- 0.012076889358565174
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_25/checkpoints/epoch=9-step=370.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_25/checkpoints/epoch=9-step=370.ckpt
model loaded


  8%|▊         | 4/50 [01:08<13:02, 17.02s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_25/checkpoints/epoch=9-step=370.ckpt accuracy on eval set: 0.730303021994504 +- 0.01406207252208932
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_52/checkpoints/epoch=4-step=185.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_52/checkpoints/epoch=4-step=185.ckpt
model loaded


 10%|█         | 5/50 [01:25<12:39, 16.87s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_52/checkpoints/epoch=4-step=185.ckpt accuracy on eval set: 0.6912878751754761 +- 0.014045366769468132
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_28/checkpoints/epoch=1-step=74.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_28/checkpoints/epoch=1-step=74.ckpt
model loaded


 12%|█▏        | 6/50 [01:41<12:13, 16.67s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_28/checkpoints/epoch=1-step=74.ckpt accuracy on eval set: 0.6751262653957714 +- 0.013699877034676322
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_55/checkpoints/epoch=7-step=296.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_55/checkpoints/epoch=7-step=296.ckpt
model loaded


 14%|█▍        | 7/50 [01:58<11:58, 16.72s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_55/checkpoints/epoch=7-step=296.ckpt accuracy on eval set: 0.7164141589945013 +- 0.010747639169412912
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_22/checkpoints/epoch=6-step=259.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_22/checkpoints/epoch=6-step=259.ckpt
model loaded


 16%|█▌        | 8/50 [02:15<11:44, 16.77s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_22/checkpoints/epoch=6-step=259.ckpt accuracy on eval set: 0.7020202008160678 +- 0.01360401729835373
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_10/checkpoints/epoch=7-step=296.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_10/checkpoints/epoch=7-step=296.ckpt
model loaded


 18%|█▊        | 9/50 [02:34<12:04, 17.67s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_10/checkpoints/epoch=7-step=296.ckpt accuracy on eval set: 0.7335858507589861 +- 0.010827443300036817
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_6/checkpoints/epoch=9-step=370.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_6/checkpoints/epoch=9-step=370.ckpt
model loaded


 20%|██        | 10/50 [03:21<17:38, 26.46s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_6/checkpoints/epoch=9-step=370.ckpt accuracy on eval set: 0.7226010127501055 +- 0.013795704721617489
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_46/checkpoints/epoch=8-step=333.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_46/checkpoints/epoch=8-step=333.ckpt
model loaded


 22%|██▏       | 11/50 [04:07<21:14, 32.68s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_46/checkpoints/epoch=8-step=333.ckpt accuracy on eval set: 0.7114898995919661 +- 0.011800106790631272
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_31/checkpoints/epoch=8-step=333.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_31/checkpoints/epoch=8-step=333.ckpt
model loaded


 24%|██▍       | 12/50 [04:52<23:00, 36.32s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_31/checkpoints/epoch=8-step=333.ckpt accuracy on eval set: 0.7313131310723044 +- 0.019427513550255234
Best model so far: accuracy: 0.7361111044883728 +- 0.008454882142943827 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_36/checkpoints/epoch=3-step=148.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
model loaded


 26%|██▌       | 13/50 [05:37<24:04, 39.04s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt accuracy on eval set: 0.7577020146630027 +- 0.012013359160620516
Best model so far: accuracy: 0.7577020146630027 +- 0.012013359160620516 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_48/checkpoints/epoch=3-step=148.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_48/checkpoints/epoch=3-step=148.ckpt
model loaded


 28%|██▊       | 14/50 [06:21<24:18, 40.51s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_48/checkpoints/epoch=3-step=148.ckpt accuracy on eval set: 0.7258838360959833 +- 0.006527357866230178
Best model so far: accuracy: 0.7577020146630027 +- 0.012013359160620516 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_42/checkpoints/epoch=11-step=444.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_42/checkpoints/epoch=11-step=444.ckpt
model loaded


 30%|███       | 15/50 [07:07<24:37, 42.23s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_42/checkpoints/epoch=11-step=444.ckpt accuracy on eval set: 0.7183080857450311 +- 0.01342287151825815
Best model so far: accuracy: 0.7577020146630027 +- 0.012013359160620516 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_35/checkpoints/epoch=4-step=185.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_35/checkpoints/epoch=4-step=185.ckpt
model loaded


 32%|███▏      | 16/50 [07:54<24:37, 43.45s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_35/checkpoints/epoch=4-step=185.ckpt accuracy on eval set: 0.6733585758642717 +- 0.012303733039244627
Best model so far: accuracy: 0.7577020146630027 +- 0.012013359160620516 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_14/checkpoints/epoch=8-step=333.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_14/checkpoints/epoch=8-step=333.ckpt
model loaded


 34%|███▍      | 17/50 [08:38<24:00, 43.66s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_14/checkpoints/epoch=8-step=333.ckpt accuracy on eval set: 0.7229798002676531 +- 0.01773382292156651
Best model so far: accuracy: 0.7577020146630027 +- 0.012013359160620516 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_51/checkpoints/epoch=5-step=222.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_51/checkpoints/epoch=5-step=222.ckpt
model loaded


 36%|███▌      | 18/50 [09:05<20:36, 38.65s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_51/checkpoints/epoch=5-step=222.ckpt accuracy on eval set: 0.7070707082748413 +- 0.01602039015751707
Best model so far: accuracy: 0.7577020146630027 +- 0.012013359160620516 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_26/checkpoints/epoch=7-step=296.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_26/checkpoints/epoch=7-step=296.ckpt
model loaded


 38%|███▊      | 19/50 [09:52<21:18, 41.23s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_26/checkpoints/epoch=7-step=296.ckpt accuracy on eval set: 0.738762617111206 +- 0.013220191318658302
Best model so far: accuracy: 0.7577020146630027 +- 0.012013359160620516 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_8/checkpoints/epoch=5-step=222.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
model loaded


 40%|████      | 20/50 [10:37<21:09, 42.30s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt accuracy on eval set: 0.7713383869691328 +- 0.015409497713505825
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_13/checkpoints/epoch=10-step=407.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_13/checkpoints/epoch=10-step=407.ckpt
model loaded


 42%|████▏     | 21/50 [10:58<17:25, 36.03s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_13/checkpoints/epoch=10-step=407.ckpt accuracy on eval set: 0.7231060591610995 +- 0.015685325817066578
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_19/checkpoints/epoch=2-step=111.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_19/checkpoints/epoch=2-step=111.ckpt
model loaded


 44%|████▍     | 22/50 [11:15<14:07, 30.28s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_19/checkpoints/epoch=2-step=111.ckpt accuracy on eval set: 0.7329545508731495 +- 0.020846790033448633
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_38/checkpoints/epoch=5-step=222.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_38/checkpoints/epoch=5-step=222.ckpt
model loaded


 46%|████▌     | 23/50 [11:33<11:55, 26.49s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_38/checkpoints/epoch=5-step=222.ckpt accuracy on eval set: 0.7377525297078219 +- 0.013025039224874533
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_32/checkpoints/epoch=5-step=222.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_32/checkpoints/epoch=5-step=222.ckpt
model loaded


 48%|████▊     | 24/50 [12:11<12:57, 29.89s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_32/checkpoints/epoch=5-step=222.ckpt accuracy on eval set: 0.7616161649877374 +- 0.010503763986813614
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_45/checkpoints/epoch=11-step=444.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_45/checkpoints/epoch=11-step=444.ckpt
model loaded


 50%|█████     | 25/50 [12:27<10:46, 25.86s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_45/checkpoints/epoch=11-step=444.ckpt accuracy on eval set: 0.7188131213188171 +- 0.014653389379207085
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_7/checkpoints/epoch=6-step=259.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_7/checkpoints/epoch=6-step=259.ckpt
model loaded


 52%|█████▏    | 26/50 [12:44<09:16, 23.20s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_7/checkpoints/epoch=6-step=259.ckpt accuracy on eval set: 0.7318181883205067 +- 0.007427964086039076
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_30/checkpoints/epoch=9-step=370.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_30/checkpoints/epoch=9-step=370.ckpt
model loaded


 54%|█████▍    | 27/50 [13:01<08:08, 21.23s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_30/checkpoints/epoch=9-step=370.ckpt accuracy on eval set: 0.7323232401501049 +- 0.013337411497372838
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_47/checkpoints/epoch=4-step=185.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_47/checkpoints/epoch=4-step=185.ckpt
model loaded


 56%|█████▌    | 28/50 [13:17<07:15, 19.80s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_47/checkpoints/epoch=4-step=185.ckpt accuracy on eval set: 0.7003787918524309 +- 0.011369372982008047
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_23/checkpoints/epoch=10-step=407.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_23/checkpoints/epoch=10-step=407.ckpt
model loaded


 58%|█████▊    | 29/50 [13:34<06:34, 18.79s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_23/checkpoints/epoch=10-step=407.ckpt accuracy on eval set: 0.692929284139113 +- 0.011898807584808085
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_54/checkpoints/epoch=13-step=518.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_54/checkpoints/epoch=13-step=518.ckpt
model loaded


 60%|██████    | 30/50 [13:50<06:01, 18.10s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_54/checkpoints/epoch=13-step=518.ckpt accuracy on eval set: 0.7147727283564481 +- 0.015230511200336772
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_29/checkpoints/epoch=11-step=444.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_29/checkpoints/epoch=11-step=444.ckpt
model loaded


 62%|██████▏   | 31/50 [14:07<05:35, 17.64s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_29/checkpoints/epoch=11-step=444.ckpt accuracy on eval set: 0.7214646556160667 +- 0.011804526413558234
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_11/checkpoints/epoch=15-step=592.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_11/checkpoints/epoch=15-step=592.ckpt
model loaded


 64%|██████▍   | 32/50 [14:23<05:10, 17.26s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_11/checkpoints/epoch=15-step=592.ckpt accuracy on eval set: 0.7162878784266385 +- 0.01322457354346569
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_16/checkpoints/epoch=11-step=444.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_16/checkpoints/epoch=11-step=444.ckpt
model loaded


 66%|██████▌   | 33/50 [14:40<04:49, 17.01s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_16/checkpoints/epoch=11-step=444.ckpt accuracy on eval set: 0.7303030274131082 +- 0.014980962499574894
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_53/checkpoints/epoch=1-step=74.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_53/checkpoints/epoch=1-step=74.ckpt
model loaded


 68%|██████▊   | 34/50 [14:57<04:33, 17.07s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_53/checkpoints/epoch=1-step=74.ckpt accuracy on eval set: 0.706313122402538 +- 0.012520218783005995
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_24/checkpoints/epoch=8-step=333.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_24/checkpoints/epoch=8-step=333.ckpt
model loaded


 70%|███████   | 35/50 [15:13<04:14, 16.98s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_24/checkpoints/epoch=8-step=333.ckpt accuracy on eval set: 0.7301767576824535 +- 0.016352925264735838
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_40/checkpoints/epoch=3-step=148.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_40/checkpoints/epoch=3-step=148.ckpt
model loaded


 72%|███████▏  | 36/50 [15:31<04:00, 17.15s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_40/checkpoints/epoch=3-step=148.ckpt accuracy on eval set: 0.7222222198139537 +- 0.01694367395569522
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_37/checkpoints/epoch=8-step=333.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_37/checkpoints/epoch=8-step=333.ckpt
model loaded


 74%|███████▍  | 37/50 [15:47<03:39, 16.91s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_37/checkpoints/epoch=8-step=333.ckpt accuracy on eval set: 0.721338385885412 +- 0.018042990622787348
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_44/checkpoints/epoch=5-step=222.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_44/checkpoints/epoch=5-step=222.ckpt
model loaded


 76%|███████▌  | 38/50 [16:03<03:18, 16.55s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_44/checkpoints/epoch=5-step=222.ckpt accuracy on eval set: 0.7630050561644814 +- 0.010949631622270297
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_33/checkpoints/epoch=3-step=148.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_33/checkpoints/epoch=3-step=148.ckpt
model loaded


 78%|███████▊  | 39/50 [16:19<03:00, 16.38s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_33/checkpoints/epoch=3-step=148.ckpt accuracy on eval set: 0.6845959587530657 +- 0.01870089801337483
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_39/checkpoints/epoch=8-step=333.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_39/checkpoints/epoch=8-step=333.ckpt
model loaded


 80%|████████  | 40/50 [16:36<02:44, 16.41s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_39/checkpoints/epoch=8-step=333.ckpt accuracy on eval set: 0.7339646382765337 +- 0.009516487758746783
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_20/checkpoints/epoch=6-step=259.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_20/checkpoints/epoch=6-step=259.ckpt
model loaded


 82%|████████▏ | 41/50 [16:52<02:27, 16.44s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_20/checkpoints/epoch=6-step=259.ckpt accuracy on eval set: 0.7262626236135309 +- 0.01075451589563588
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_18/checkpoints/epoch=3-step=148.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_18/checkpoints/epoch=3-step=148.ckpt
model loaded


 84%|████████▍ | 42/50 [17:09<02:11, 16.48s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_18/checkpoints/epoch=3-step=148.ckpt accuracy on eval set: 0.701136356050318 +- 0.011415172786154174
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_12/checkpoints/epoch=8-step=333.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_12/checkpoints/epoch=8-step=333.ckpt
model loaded


 86%|████████▌ | 43/50 [17:26<01:56, 16.69s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_12/checkpoints/epoch=8-step=333.ckpt accuracy on eval set: 0.7224747484380548 +- 0.016839684914718007
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_15/checkpoints/epoch=7-step=296.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_15/checkpoints/epoch=7-step=296.ckpt
model loaded


 88%|████████▊ | 44/50 [17:42<01:39, 16.64s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_15/checkpoints/epoch=7-step=296.ckpt accuracy on eval set: 0.7236111109906976 +- 0.013100926872012976
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_27/checkpoints/epoch=7-step=296.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_27/checkpoints/epoch=7-step=296.ckpt
model loaded


 90%|█████████ | 45/50 [17:59<01:23, 16.69s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_27/checkpoints/epoch=7-step=296.ckpt accuracy on eval set: 0.7203282930634238 +- 0.008523002305362128
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_50/checkpoints/epoch=3-step=148.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_50/checkpoints/epoch=3-step=148.ckpt
model loaded


 92%|█████████▏| 46/50 [18:15<01:06, 16.59s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_50/checkpoints/epoch=3-step=148.ckpt accuracy on eval set: 0.748611097986048 +- 0.011352409220268004
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_9/checkpoints/epoch=11-step=444.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_9/checkpoints/epoch=11-step=444.ckpt
model loaded


 94%|█████████▍| 47/50 [18:32<00:49, 16.62s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_9/checkpoints/epoch=11-step=444.ckpt accuracy on eval set: 0.7290404059670188 +- 0.015280860985669402
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_34/checkpoints/epoch=9-step=370.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_34/checkpoints/epoch=9-step=370.ckpt
model loaded


 96%|█████████▌| 48/50 [18:49<00:33, 16.74s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_34/checkpoints/epoch=9-step=370.ckpt accuracy on eval set: 0.7218434322964061 +- 0.01316053070201387
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_43/checkpoints/epoch=12-step=481.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_43/checkpoints/epoch=12-step=481.ckpt
model loaded


 98%|█████████▊| 49/50 [19:06<00:16, 16.65s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_43/checkpoints/epoch=12-step=481.ckpt accuracy on eval set: 0.7207070643251593 +- 0.01507921924171158
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt
Testing model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_49/checkpoints/epoch=2-step=111.ckpt
Initialized mask embedding and position encoder from  /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_49/checkpoints/epoch=2-step=111.ckpt
model loaded


100%|██████████| 50/50 [19:21<00:00, 23.24s/it]

Model /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_49/checkpoints/epoch=2-step=111.ckpt accuracy on eval set: 0.6582070751623674 +- 0.011605801416592875
Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt


### on eval set:

Best model so far: accuracy: 0.6795454545454546 +- 0.013922615366108731 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_4/version_9/checkpoints/epoch=14-step=150.ckpt

Best model so far: accuracy: 0.768560604615645 +- 0.011691771714137506 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset/version_47/checkpoints/epoch=2-step=141.ckpt

Best model so far: accuracy: 0.7713383869691328 +- 0.015409497713505825 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_21/checkpoints/epoch=6-step=259.ckpt

### on test set:

Best model so far: accuracy: 0.979166673289405 +- 0.002758420816857501 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset/version_1/checkpoints/epoch=13-step=658.ckpt



### on split eval set

Best model so far: accuracy: 0.8546875069538752 +- 0.005056377527444972 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_39/checkpoints/epoch=8-step=333.ckpt


### on split test set

Best model so far: accuracy: 0.9757505257924398 +- 0.002019294366371989 | /scratch/s194101/finetune_logs_all/tusz_noica_noica_60.0_nogroups_whole_dataset_split_train/version_54/checkpoints/epoch=13-step=518.ckpt

# MMIDB data